src/components/SettingDialog.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <q-dialog
+    maximized
+    persistent
+    class="setting-dialog"
+    v-model="settingDialogOpenedComputed"
+    transition-show="slide-up"
+    transition-hide="slide-down"
+  >
+    <q-layout container view="hHh Lpr fFf" class="bg-white">
+      <q-header class="q-pa-sm" elevated>
+        <q-toolbar>
+          <q-toolbar-title class="text-secondary">オプション</q-toolbar-title>
+          <q-space />
+          <!-- colse button -->
+          <q-btn
+            round
+            flat
+            icon="close"
+            @click="settingDialogOpenedComputed = false"
+          />
+        </q-toolbar>
+      </q-header>
+      <q-page-container>
+        <q-page ref="scroller" class="relative-absolute-wrapper scroller">
+          <div class="q-pa-md row items-start q-gutter-md">
+            <!-- Engine Mode Card -->
+            <q-card class="setting-card">
+              <q-card-section class="bg-nvidia">
+                <div class="text-h5">エンジン</div>
+                <div class="text-subtitle2">
+                  GPUモードには <b>NVIDIA&trade;</b> GPUが必要です
+                </div>
+              </q-card-section>
+
+              <q-separator />
+              <q-card-actions class="q-px-md">
+                <q-radio
+                  v-model="engineMode"
+                  color="green"
+                  val="switchCPU"
+                  label="CPU"
+                />
+                <q-radio
+                  v-model="engineMode"
+                  color="green"
+                  val="switchGPU"
+                  label="GPU"
+                />
+                <div class="q-pl-lg">
+                  <q-btn
+                    size="12px"
+                    label="再起動"
```

Comment:
最後に１つだけお願いします！
メニューにエンジン再起動ボタンがあるのと、GPUモード変更時に自動でエンジンが再起動されるので、このボタンはなくしましょう。

---

One last thing, please!
Since there is an engine restart button in the menu and the engine is automatically restarted when changing GPU mode, so please remove this button.